README.rst

```diff
@@ -1,2 +1,56 @@
 Pillow Website
 ==============
+
+This website is live at `python-pillow.org <https://python-pillow.org>`_
+
+Website structure:
+------------------
+
+.. code-block:: text
+
+   ├── assets
```

Comment:
This directory listing feels redundant and is likely to go out of sync. Can we remove it?

Reply:
Sure, I'll remove it